build_tools/python/e2e_model_tests/test_definitions.py

```diff
@@ -18,10 +18,10 @@
 
 class CMakePlatform(enum.Enum):
   """Enum of CMake system platform string."""
-  ANDROID_ARMV8_A = "android-arm64-v8a"
-  LINUX_RISCV32 = "riscv32-Linux"
-  LINUX_RISCV64 = "riscv64-Linux"
-  LINUX_X86_64 = "x86_64"
+  ANDROID_ARMV8_A = "arm_64-Android"
+  LINUX_RISCV32 = "riscv_32-Linux"
+  LINUX_RISCV64 = "riscv_64-Linux"
+  LINUX_X86_64 = "x86_64-Linux"
```

Comment:
Maybe rename these to match?

```suggestion
  ARM_64_ANDROID = "arm_64-Android"
  RISCV_32_LINUX = "riscv_32-Linux"
  RISCV_64_LINUX = "riscv_64-Linux"
  X86_64_LINUX = "x86_64-Linux"
```

Reply:
Sorry I missed your comments here.